[2020-12-25 17:15:56,476] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: foodpanda.docker_op_tester 2020-12-25T09:15:52.113061+00:00 [queued]>
[2020-12-25 17:15:56,479] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: foodpanda.docker_op_tester 2020-12-25T09:15:52.113061+00:00 [queued]>
[2020-12-25 17:15:56,479] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2020-12-25 17:15:56,479] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2020-12-25 17:15:56,479] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2020-12-25 17:15:56,488] {taskinstance.py:1038} INFO - Executing <Task(DockerOperator): docker_op_tester> on 2020-12-25T09:15:52.113061+00:00
[2020-12-25 17:15:56,489] {standard_task_runner.py:51} INFO - Started process 503 to run task
[2020-12-25 17:15:56,492] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'foodpanda', 'docker_op_tester', '2020-12-25T09:15:52.113061+00:00', '--job-id', '50', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ssh_practice.py', '--cfg-path', '/tmp/tmpbk_th58u']
[2020-12-25 17:15:56,492] {standard_task_runner.py:76} INFO - Job 50: Subtask docker_op_tester
[2020-12-25 17:15:56,508] {logging_mixin.py:103} INFO - Running <TaskInstance: foodpanda.docker_op_tester 2020-12-25T09:15:52.113061+00:00 [running]> on host dataiku
[2020-12-25 17:15:56,522] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=arthur.sl.lin
AIRFLOW_CTX_DAG_ID=foodpanda
AIRFLOW_CTX_TASK_ID=docker_op_tester
AIRFLOW_CTX_EXECUTION_DATE=2020-12-25T09:15:52.113061+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2020-12-25T09:15:52.113061+00:00
[2020-12-25 17:15:56,526] {docker.py:224} INFO - Starting docker container from image hello-world
[2020-12-25 17:15:56,748] {docker.py:263} INFO - Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (amd64)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/
[2020-12-25 17:15:56,955] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=foodpanda, task_id=docker_op_tester, execution_date=20201225T091552, start_date=20201225T091556, end_date=20201225T091556
[2020-12-25 17:15:56,967] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2020-12-25 17:15:56,985] {local_task_job.py:118} INFO - Task exited with return code 0
